gammapy/data/counts.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import print_function, division
+from gammapy.spectrum import EnergyBinning
+
+
+__all__ = ['CountsSpectrumDataset',
+           ]
+
+
+class CountsSpectrumDataset(object):
+    """
+    Counts spectrum dataset (PHA wrapper).
+    
+    Test case for EnergyBinning class
+    """
+  
+    def __init__(self, counts, energy):
+
+        #does not work (why?) -> Kind of crucial
```

Comment:
The Quantity verification looks useful. I guess there's no point in merging this, until we agreed if it is a good idea or not